govdelivery-subscribe/error/index.html

```diff
@@ -9,15 +9,16 @@
 {%- endblock %}
 
 {% block content_main %}
-    <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
+    <h1><span class="cf-icon cf-icon-error-round"></span> That didn't work.</h1>
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try <br>
+        to submit your address again in a little while. For <br>
```

Comment:
Should the line breaks be at the end of the sentences? Like: 

```html
We couldn’t complete your signup request. <br>
Try to submit your address again in a little while. <br>
For now:
```

Or is that existing line breaks intentional?